docs/iris/src/whatsnew/3.0.rst

```diff
@@ -1,12 +1,44 @@
 .. include:: ../common_links.inc
 
-v3.0 (01 Oct 2020)
+v3.0 (02 Oct 2020)
 ******************
 
 This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
+.. dropdown:: :opticon:`report` Release Highlights
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+
+   The highlights for this major release of Iris include:
+
+   * We've finally dropped support for ``Python 2``, so welcome to ``Iris 3``
+     and ``Python 3``!
+   * Our :ref:`documentation <whatsnew 3.0 docs>` has been refreshed,
+     restructured, revitalised and rehosted on `readthedocs`_,
+   * It's now easier than ever to :ref:`install Iris <installing_iris>`
+     as a user or a developer, and the developers guide walks you though
```

Comment:
how about "newly revamped developers guide" or similar?  I don't think the improvement can be overstated...